js/pages/concept-sets/concept-manager.html

```diff
@@ -68,94 +68,105 @@
+										{ title: 'DRC', data: 'DESCENDANT_RECORD_COUNT'},
+										{ title: 'Distance', data: function (d) { return String(Math.min.apply(Math, d.RELATIONSHIPS.map(function (o) {return o.RELATIONSHIP_DISTANCE;})))}},
+										{ title: 'Domain', data: 'DOMAIN_ID' },
+										{ title: 'Vocabulary', data: 'VOCABULARY_ID', width: '100px' }
+								],
+								options: hierarchyConceptsOptions,
+								reference: $component.metarchy.parents, 
+								rowCallback: commonUtils.contextSensitiveLinkColor 
+							}"></faceted-datatable>
+						</div>
+					</div>
                 </div>
-                <div id="wrapperHierarchyAnchor">
-                    <table width="100%" class="conceptTable stripe compact hover" cellspacing="0" data-bind="dataTable:{
-                                            data: currentConceptArray,
-                                            options: {
-                                                dom: 't',
-                                                language: {
-                                                    search: 'Filter Broader Concepts: '
-                                                },
-                                                rowCallback: commonUtils.contextSensitiveLinkColor,
-                                                lengthMenu: [[-1], ['All']],
-                                                orderClasses: false,
-                                                ordering: true,
-                                                order: [[ 6, 'desc' ]],
-                                                columns: [
-                                                        { title: 'Id', data: 'CONCEPT_ID'},
-                                                        { title: 'Code', data: 'CONCEPT_CODE'},
-                                                        { title: 'Name', data: 'CONCEPT_NAME', render: commonUtils.renderLink, width: '50%'},
-                                                        { title: 'Class', data: 'CONCEPT_CLASS_ID' },
-                                                        { title: 'RC', data: 'RECORD_COUNT'},
-                                                        { title: 'DRC', data: 'DESCENDANT_RECORD_COUNT'},
-                                                        { title: 'Distance', data: function (d) { return String(Math.min.apply(Math, d.RELATIONSHIPS.map(function (o) {return o.RELATIONSHIP_DISTANCE;})))}},
-                                                        { title: 'Domain', data: 'DOMAIN_ID' },
-                                                        { title: 'Vocabulary', data: 'VOCABULARY_ID', width: '100px' }
-                                                ]
-                                        }
-                                    }">
-                    </table>
+                <div id="wrapperHierarchyAnchor" data-bind="visible: $component.hierarchyPillMode() == 'all' || $component.hierarchyPillMode() == 'current'">
```

Comment:
Suggestion: `['all', 'current'].includes($component.hierarchyPillMode())`

Reply:
Tried this as: `visible: ['all', 'children'].includes($component.hierarchyPillMode())` as suggested but it appears the knockout binding does not respond to changes in the observable when it is nested inside of the `includes` function. So, I'll leave for now unless you have another suggestion on this one.